downgrading.md

```diff
@@ -0,0 +1,38 @@
+## Automatically using `ipfs-update`
+
+If you want to do a downgrade (and not a clean install of an older version) you can revert your repo using the fs-migration tools:
+
+```sh
+$ ipfs-2-to-3 -revert -path=$IPFS_PATH
+```
+
+That should be it. You're all set.
+
+## Manually downgrading IPFS versions
+
+**You should not have to do this manually. Automatically downgrading is the preferred method.**
+
+Manually downgrading your IPFS version (for instance, from `0.4.0` to `0.3.8`) is not as complicated as upgrading. You can't use the same repository simultaneously for both versions. Here is how you switch. Note that this is actually a clean install, and that your daemon won't have the same data as before.
```

Comment:
I feel like we make that clear? But I've pushed a slightly different version making this blatant.